math-dojo/admin-web-ui
# Ft1

## Purpose

* Adds the component to approve questions from azure queue to admin ui
* Adds a home page and actions page and their respective components to the admin UI
* Adds functionality to approve quarantined questions from azure queue store using functions




## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code
npm start
go to link http://localhost:4200/actions/approve
check that admin UI can approve quarantined questions
